kurtosis-devnet/interop.yaml

```diff
@@ -32,6 +32,11 @@ optimism_package:
               "chainIndex": "2151909",
               "activationTime": 0,
               "historyMinTime": 0
+            },
+            "2151910": {
+              "chainIndex": "2151910",
+              "activationTime": 0,
+              "historyMinTime": 0
```

Comment:
Does it not work with just 2 chains?

While I think 3 chains is great, the kurtosis setup is getting quite compute/memory-heavy, and so I think it's better to not make 3 chains the new default.

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/15049/commits/524a9971abb0761eeff357aee8c221066f289d29